jenkins-pipelines/manager/debian9-manager-upgrade.jenkinsfile

```diff
@@ -11,8 +11,8 @@ managerPipeline(
     target_scylla_mgmt_server_repo: 'http://downloads.scylladb.com/manager/deb/unstable/stretch/master/latest/scylladb-manager-master/scylla-manager.list',
     target_scylla_mgmt_agent_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
 
-    scylla_mgmt_repo: 'http://downloads.scylladb.com/manager/deb/unstable/stretch/master/latest/scylladb-manager-master/scylla-manager.list',
-    scylla_mgmt_agent_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
```

Comment:
@fruch this was the issue with the upgrade test of debian.

Reply:
good